sdk_commons/pyproject.toml

```diff
@@ -12,6 +12,7 @@ authors = [
 ]
 packages = [
     { include = 'sdk_commons' },
+    { include = 'sdk_commons/abis' },
```

Comment:
Isn't this supposed to be `'abis'` only?

Reply:
No, in the end I put abis into sdk_commons to access the files easier from the get_abi helper